apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -38,10 +61,16 @@ const AnimationTab = React.createClass({
   propTypes: {
     channelId: React.PropTypes.string.isRequired,
     columnSizes: React.PropTypes.arrayOf(React.PropTypes.number).isRequired,
-    onColumnWidthsChange: React.PropTypes.func.isRequired
+    onColumnWidthsChange: React.PropTypes.func.isRequired,
+    // Provided by Redux
+    selectedAnimation: PropTypes.AnimationKey
```

Comment:
Thanks for annotating.  While you're at it, `columnSizes` is also provided by Redux (see `connect` method at end of file).